fun main() {
    // Write some code to cause an exception
    val array = arrayOf(1)
    println(array[2])
}